src/OLEDDisplay.h

```diff
@@ -160,7 +160,10 @@ class OLEDDisplay : public Stream {
 	uint16_t width(void) const { return displayWidth; };
 	uint16_t height(void) const { return displayHeight; };
 
-    // Initialize the display
+    // Prepare internal structures for resuming display usage after a deep sleep.
```

Comment:
Please describe here when it returns true and when the return value is false, thanks.